dataland-internal-storage/src/main/kotlin/org/dataland/datalandinternalstorage/services/ConfigurationBackendControllerApi.kt

```diff
@@ -0,0 +1,24 @@
+package org.dataland.datalandinternalstorage.services
+
+import org.dataland.datalandbackend.openApiClient.api.NonPersistedDataControllerApi
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+/**
+ * This class holds the configuration properties and the beans to auto-configure this library in a spring-boot
+ * environment
+ */
+@Configuration
+class ConfigurationBackendControllerApi {
+    /**
+     * The bean to configure the internal client StorageControllerApi
+     */
+    @Bean
+    fun getApiBackendClient(): NonPersistedDataControllerApi {
+        val backendContainerUrl = "http://backend:8080/api"
+        val internalBackendUrlFromEnv = System.getenv("INTERNAL_BACKEND_URL")
+        return NonPersistedDataControllerApi(
+            internalBackendUrlFromEnv?.ifBlank { backendContainerUrl } ?: backendContainerUrl,
```

Comment:
why not use application.properties - with 2 profiles (development and "default")?

To my knowledge, we didn't configure Spring-Services using Environment Variables so far.

Reply:
Possible. Doing as you describe felt like a bit more work. It would induce another properties file and a new dockerfile for the internal storage service. Can be done though, if you prefer it. I'm also not much of a fan of the environment variable solution, but thought it was ok because there is no env to be defined when all the services run in a docker container which holds true for almost all environments.

Reply:
Wie besprochen: 
-> Auslagern in Application.property. Da kann dann das Env-Variablen-Parsing stattfinden
-> Im Refactoring Log das lgeiche für den email manager eintragen.

Reply:
1. Done, 2. Should this even be done. It really would be only a wrapper for an env at one point in the code

Reply:
1. ok
2. Können wir ja im Team diskutieren. Ich finde einheitlich schon besser.

Reply:
Done

Reply:
ok